entrypoint.sh

```diff
@@ -81,8 +81,7 @@ case "$format" in
     fi
   ;;
   *)
-    echo "format $format not yet implemented."
-    exit 2
+    echo "format $format not yet officially supported within action."
```

Comment:
Most package formats will work without requiring any significant modifications to this action. Instead of requiring a format to be added to `master` before it can be tested; let's flag that it's not officially supported yet within the action. 

This should also make contributing new formats much easier and only require a single PR to add a format, documentation, and a working example.